File: 프로그래머스/0/181902. 문자 개수 세기/문자 개수 세기.js
function solution(my_string) {
  const capital = Array(26).fill().map((v, i) => String.fromCharCode(i + 65));
  const small = capital.map(x => x.toLowerCase())
  const alphabet = [...capital, ...small];
  let result = Array(52).fill(0);

  for (let i = 0; i < my_string.length; i++) {
    const arr = alphabet.filter(x => x === [...my_string][i]);
    const index = alphabet.indexOf([...my_string][i])
    result[index] += arr.length
  }
  return result
}

File: 프로그래머스/0/181902. 문자 개수 세기/문자 개수 세기.java
class Solution {
      public int[] solution(String my_string) {
        int[] result = new int[52];
        
        for (char c : my_string.toCharArray()) {
            if (c >= 'A' && c <= 'Z') {
                result[c - 'A']++;
            } else if (c >= 'a' && c <= 'z') {
                result[c - 'a' + 26]++;
            }
        }
        
        return result;
    }
}